### ISSUE TITLE
The definition of trial_period for prices is incorrect

### ISSUE THREAD

**Issue by archatas** [websideproject/paddle-billing-client#160]
## 🐛 Bug Report

Trying to list prices with any of the prices with a trial period, raises a validation error.

## 🔬 How To Reproduce

1. Create a price with a trial period at Paddle Billing
2. Try to list prices with `client.list_prices()`

### Code sample

The definition of `trial_period` must be an optional `TrialPeriod` object.

Wrong:

```python
class PriceBase(BaseModel):
    trial_period: Optional[str]
```

Probably right (untested):

```python
class TrialPeriod(BaseModel):
    interval: str
    frequency: int

class PriceBase(BaseModel):
    trial_period: Optional[TrialPeriod]
```

### Environment

* OS: macOS
* Python 3.10

**Comment by archatas** [comment]
Here is the original trial_period definition:
https://developer.paddle.com/api-reference/prices/overview

**Comment by bgervan** [comment]
Fixed with https://github.com/websideproject/paddle-billing-client/pull/163

**Comment by bgervan** [comment]
Released in 0.1.13

**Comment by archatas** [comment]
Thank you for such a quick reaction!
